### PR TITLE
Usagov 394 telephone number for mobile

### PR DESCRIPTION
**Pull request by IsabelLaurenceau**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-394

## Checklist for the Developer

- [x] A link to the JIRA ticket has been added above.
- [x] The JIRA ticket identifies the desired result of this change.
- [x] The JIRA ticket contains clear acceptance criteria.
- [x] The JIRA ticket contains clear testing steps.
- [ ] The JIRA ticket contains a description of "Done".
- [x] Any preparation/installation/update steps required for this code to work properly (drush commands, scripts, configuration updates) are documented in the ticket.

## Checklist for the Peer Reviewers
- [x] The branch name of this PR matches the project standards.
- [x] QA steps are followed and any changes to process are updated in the ticket.
- [x] Code Standards are followed, there are no bad practices in use.
- [x] Config changes (if any) include only necessary changes.
- [x] No errors in Drupal or Client Browser.
- [x] Code has been tested locally (if possible).
- [x] Following AC and Testing Steps verify changes work as expected.
- [x] There are no known side-effects outside of expected behavior.

### CODE REVIEW

File: web/themes/custom/usagov/templates/node--directory-record--full.html.twig [hunk 1]
```diff
@@ -14,21 +14,31 @@
   {% if content.field_phone_number|render %} 
   <div>
     <h3>Phone Number</h3>
-    <p>{{content.field_phone_number}}</p>
+    <p>
+
```
Comment (akf) [inline web/themes/custom/usagov/templates/node--directory-record--full.html.twig]: Just curious -- is there a reason for the additional blank lines here?

File: web/themes/custom/usagov/scripts/telephoneForMobilePage.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+(function ($) {
```
Comment (akf) [inline web/themes/custom/usagov/scripts/telephoneForMobilePage.js]: It looks like there are two versions of this script, and this one is not used -- if so, let's delete it.

File: web/themes/custom/usagov/scripts/telephoneForMobileNode.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+jQuery(document).ready(function ($) {
+  if (window.innerWidth <= 950) {
+    $(".field--type-telephone").replaceWith(function () {
+      return $(
+        "<a href='tel:" + $(this).html() + "'>" + $(this).html() + "</a>"
+      );
+    });
+  }
+
+  if (window.innerWidth >= 950) {
```
Comment (akf) [inline web/themes/custom/usagov/scripts/telephoneForMobileNode.js]: From what I'm seeing, we don't have anything triggering this script when the viewport changes, so I don't believe there's any need to "reset" the links.

**Review by akf — CHANGES_REQUESTED** [review]
This is doing a nice job of linking the phone numbers when I use a narrow viewport. 

One thing you missed: content editors will enter something like "800-555-1234 (Central Dispatch)" in a phone field, and we want just the "800-555-1234" bit to be linked. They've been doing this on the current mothership so I want to support that, since we'll be importing all that data! There are some notes in Jira but I'm going to flesh them out a little.

File: web/themes/custom/usagov/scripts/telephoneForMobileNode.js (hunk 3 above)
Comment (akf) [reply web/themes/custom/usagov/scripts/telephoneForMobileNode.js]: ... which I now notice was in the example from mothership, but I guess I wasn't in review mode when I looked at that!

File: web/themes/custom/usagov/templates/node--directory-record--full.html.twig (hunk 1 above)
Comment (IsabelLaurenceau) [reply web/themes/custom/usagov/templates/node--directory-record--full.html.twig]: No just my vscode settings, I'll remove them!

File: web/themes/custom/usagov/scripts/telephoneForMobilePage.js (hunk 2 above)
Comment (IsabelLaurenceau) [reply web/themes/custom/usagov/scripts/telephoneForMobilePage.js]: Will do!

File: web/themes/custom/usagov/scripts/telephoneForMobileNode.js (hunk 3 above)
Comment (IsabelLaurenceau) [reply web/themes/custom/usagov/scripts/telephoneForMobileNode.js]: I'll update this as well

**Comment by IsabelLaurenceau** [comment]
I have updated the code to use the methodology of mothership. Meaning I get the number first by using a space as the split. Than I clean the numbers with Regex and display/link it appropriately.

**Comment by IsabelLaurenceau** [comment]
Will wait until meeting to request review. May update depending on information

**Review by akf — APPROVED** [review]
Looks good and works for me. I have an idea for a follow-on task, but this is good to go.

**Pull request merged**